config.sh

```diff
@@ -57,7 +61,7 @@ JQ_FILE="$JQ_DIR.tar.gz"
 JQ_URL="https://github.com/stedolan/jq/releases/download/jq-$JQ_VERSION/$JQ_FILE"
 JQ_OUT="jq"
 
-NPM_CLI="$BUILD_NAME/lib/node_modules/npm/bin/npm-cli.js"
+NPM_CLI="lib/node_modules/npm/bin/npm-cli.js"
 
 if [ "$(uname -s)" == "Darwin" ]; then
```

Comment:
not connected with the PR, but next 2 checks are repeated and can be merged into 1

Reply:
check was removed.